user.tf

```
@@ -122,6 +122,10 @@ data "github_user" "r1kkert" {
   username = "r1kkert"
 }
 
+data "github_user" "plassche-dictu" {
+  username = "plassche-dictu"
```

```suggestion
  username = "PLassche-dictu"
```

Zou je er dit van kunnen maken?